sdcm/results_analyze/test.py

```diff
@@ -475,11 +476,16 @@ def _get_es_query_from_self(self, data_path_patterns: list):
     def get_by_params(cls, es_index=es_index, **params):
         es_query = cls._get_es_query_from_instance_data(params)
         filter_path = cls._get_es_filters()
-        es_data = ES().search(
-            index=es_index,
-            q=es_query,
-            filter_path=filter_path,
-            size=10000)
+        try:
+            es_data = ES().search(
+                index=es_index,
+                q=es_query,
+                filter_path=filter_path,
+                size=10000)
+        except Exception as exc:  # pylint: disable=broad-except
+            LOGGER.warning("Unable to find ES data: {exc}".format(exc=exc))
```

Comment:
use lazy interpolation
https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation

Reply:
As we found out, outdated pylint version doesn't support union type hints, fixed and removed

Reply:
done

Reply:
I don't see the change

Reply:
corrected for real this time, somehow missed both